sklearn/metrics/tests/test_classification.py

```diff
@@ -103,7 +103,6 @@ def make_prediction(dataset=None, binary=False):
 
 
 def test_classification_report_dictionary_output():
-
```

Comment:
I suspect this is because of a newer version of `black`. Maybe it's time to update `black` for the repo.

Reply:
Uhm this is the pre-commit version. I don't install black otherwise.